### PR TITLE
Update: Template Browser Collections Page Category Selection

### PR DESCRIPTION
**Pull request by jennsun**
https://app.clickup.com/t/13qkz2w

Updated layout of the page to have category names listed on the left with two columns of video data on the right instead of an expanded list. Removed aspect ratio filtering so videos of all aspect ratios show up and video thumbnail boxes fit according to respective aspect ratios.

### CODE REVIEW

File: example_site/src/components/Collections.css [hunk 1]
```diff
@@ -2,32 +2,35 @@
...
-ul.collection {
-  margin-top: 10px;
-  list-style: none;
-  display: grid;
-  grid-template-columns: repeat(8, 1fr);
-  gap: 10px;
+.categories {
+  position: relative;
+  left: 30%;
+  width: 300px;
```
Comment (simanthastew) [inline example_site/src/components/Collections.css]: You should not need to set the width on your grid items! You can use a combination of the [grid column](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column) and gap settings.

[This is a nice code sandbox](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column) to play around with grid layouts to get a better understanding of how all of these work!

File: example_site/src/components/Collections.js [hunk 2]
```diff
@@ -12,23 +12,26 @@ function Template({ template }) {
...
   return (
-    <li className="template">
-      <button title={template.id} onClick={() => openEditor({ template })}>
-        <img
-          className="thumbnail"
-          src={template.thumbnailImageURL}
-          alt={`${template.name} thumbnail`}
-        />
-        <div>{template.name}</div>
+    <div>
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: Do you need this surrounding `div`?

File: example_site/src/components/Collections.js [hunk 3]
```diff
@@ -37,24 +40,9 @@ function Collection({
...
-
-  useEffect(() => {
-    if (!_.isEmpty(templates)) {
-      addTemplates(templates);
-    }
-  }, [templates, addTemplates]);
-
   const onClick = (collection) => {
     setSelectedCollection(collection);
+    console.log("Clicked");
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: Let's get rid of this log statement

File: example_site/src/components/Collections.js [hunk 4]
```diff
@@ -12,23 +12,26 @@ function Template({ template }) {
...
+        <div className='image-container'>
+            <img
+              className="thumbnail"
+              src={template.thumbnailImageURL}
+              alt={`${template.name} thumbnail`}
+            />
+        </div>
+        <div>Name: {template.name}</div>
+        <div>Aspect Ratio: {template.aspectRatio}</div>
+        <div>Duration: {template.duration}</div>
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: Semantically speaking, this should probably be a list! I really liked [this article](https://seekbrevity.com/semantic-markup-important-web-design/) about why respecting HTML semantics is important. 

```
<ul>
    <li>Name: {template.name}</li>
    ....
</ul>
```

Updating that will require some [additional styling](https://www.w3schools.com/css/css_list.asp) to remove the default list styles.

File: example_site/src/components/Collections.js [hunk 5]
```diff
@@ -63,42 +51,74 @@ function Collection({
...
   return (
-    <li className="collection">
+    <div className="category-name">
       <button className="collection-name" onClick={() => onClick(collection)}>
-        {collection.name} ({collection.id})
+            {collection.name}
       </button>
-      {shouldShowRefetchWithFilterButton && (
+
+      {shouldShowRefetchWithFilterButton ? (
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: Are we still using this value and button?

File: example_site/src/components/Collections.js [hunk 6]
```diff
@@ -117,27 +137,37 @@ export default function Collections() {
...
-                  selectedCollection && selectedCollection.id === collection.id
-                )}
-                templateFilter={templateFilter}
+                />
+              ))}
+            </div>
+            <div className="templates">
+              <CollectionTemplates
+                collection ={selectedCollection}
+                expand={selectedCollection != null}
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: I don't think we need to pass down this `expand` prop anymore.

`CollectionTemplates` handles its own loading/error state, and the `expand` value is just used to render the HTML or not, so you could rewrite this like so:
```
{selectedCollection ? (
    <CollectionTemplates .../>
: null)
```

File: example_site/src/components/Collections.js [hunk 7]
```diff
@@ -117,27 +137,37 @@ export default function Collections() {
...
-              <Collection
-                key={collection.id}
-                collection={collection}
+        <JsonEditor value={templateFilter} onChange={setTemplateFilter} /> */}
+          <div className="browser-columns">
+            <div className="categories">
+              <h2>Categories</h2>
+              {collections.map((collection) => (
+                <CollectionNames
+                collection ={collection}
```
Comment (simanthastew) [inline example_site/src/components/Collections.js]: This line and the line below need to be indented one more time

**Review by simanthastew — CHANGES_REQUESTED** [review]
Nice work! Just a little bit of cleanup :)

File: example_site/src/components/Collections.css (hunk 1 above)
Comment (jennsun) [reply example_site/src/components/Collections.css]: sandbox: https://codepen.io/discodavey/pen/aJXgbq

**Review by simanthastew — CHANGES_REQUESTED** [review]
Changes look great! Last thing would be having the selected collection name be blue instead of black so the user knows which collection is selected.

**Pull request merged**